tasks/Debian.yml

```diff
@@ -37,3 +37,10 @@
   notify:
     - restart collectd
     - stackdriver updated
+
+- name: "[Debian] Install mysql-client if needed for mysql plugin."
+  apt: name=mysql-client state=present
```

Comment:
This plugin may connect with a distant host/container. so mysql-client might not be installed (like in gcloud debian images). And since the requirement is in the README, it seems coherent to add it there.